test/integration/script.py

```diff
@@ -107,7 +107,8 @@ def points_to_string(points):
 
 
 def check_file_annotation(client, file_annotation,
-                          parent_type="Image", is_linked=True):
+                          parent_type="Image", is_linked=True,
```

Comment:
This change will have to be propagated to https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/testlib/script.py

Reply:
Ported in https://github.com/openmicroscopy/openmicroscopy/pull/5737